kms/asymmetric/samples.go

```diff
@@ -0,0 +1,189 @@
+package main
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/asn1"
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"golang.org/x/net/context"
+	"google.golang.org/api/cloudkms/v1"
+	"log"
+	"math/big"
+
+	"golang.org/x/oauth2/google"
+)
+
+// [START kms_get_asymmetric_public]
+// Retrieve a public key from a saved asymmetric key pair on KMS
```

Comment:
godoc format begins with the method name.

    // getAsymmetricPublicKey does x y z.

add a trailing comma.

Also see if other samples export these helper functions or not.

Reply:
and you'll want to put a newline between the doc string and the region tag.